src/webgpu/web_platform/external_texture/video.spec.ts

```diff
@@ -99,17 +127,16 @@ g.test('importExternalTexture,sample')
   .desc(
     `
 Tests that we can import an HTMLVideoElement into a GPUExternalTexture, sample from it for all
-supported video formats {vp8, vp9, ogg, mp4}, and ensure the GPUExternalTexture is destroyed by
-a microtask.
-TODO: Multiplanar scenarios
+supported video formats {vp8, vp9, ogg}, common source colorspaces {bt.601, bt.709, bt.2020}, 
+and to ensure the GPUExternalTexture is destroyed by a microtask.
 `
   )
   .params(u =>
     u //
-      .combine('videoSource', kVideoSources)
+      .combine('videoExpectations', kVideoExpectations)
```

Comment:
This adds the expectations to the query string, which is unnecessary. You can get them passed through with a "private" param which is a param that starts with `_`. I'll do this in a commit that I'm pushing to your branch.